docs/hpc/jobs.md

```diff
@@ -39,6 +39,7 @@ The file below represents a sample Slurm script where a conda base environment i
 #SBATCH --nodes=1                	# node count
 #SBATCH --ntasks=1               	# total number of tasks across all nodes
 #SBATCH --time=00:10:00          	# total run time limit (HH:MM:SS)
+#SBATCH --account=iacc_gbuzzell		# SLURM account name
```

Comment:
Good addition!  Can we add to the table, as well, with explanation of when this should be used (and that if we aren't using the high mem queue that you simply delete the whole line)?

Reply:
Yeah if you don't need high-mem this line isn't needed